street2sat_utils/crop_info/instructions_for_labeling.md

```diff
@@ -0,0 +1,16 @@
+## Instructions for Labeling Images
+
+1. Go to this labeling tool online: https://www.makesense.ai/.
+2. Click get started
+3. Click drop images and select the images you wish to label. Select multiple by holding down the select key.
+4. Click Object Detection
+5. Click Load Labels From File
+6. Upload the classes.txt file
```

Comment:
So should this file be available in the repository?

Reply:
Thats a good idea, I'll drop it in the same folder.